### DISCUSSION TITLE
目前的版本可以对接到nginx吗？

### DISCUSSION THREAD

**Discussion by xxwxgz** [alibaba/xquic#77] [Q&A] [answered]
我看介绍材料有对接到nginx，这个有开放吗？怎么对接？

服务端方面，由于阿里内部网关体系广泛使用 Tengine（Nginx 开源分支），我们开发了一个 ngx_xquic_module 用于适配 Tengine 服务端。

**Comment by Kulsk** [comment, marked as answer]
we do integrated xquic with tengine as a mudule, and the module will also be opened in future
